src/Hyperion/Main.hs

```diff
@@ -130,22 +134,26 @@ doRun bks = do
       throwIO $ DuplicateNames [ n | n:_:_ <- group (sort nms) ]
     foldMap (runBenchmark (sample 100 (fixed 5))) bks
 
-doAnalyze :: Config -> [Benchmark] -> IO ()
-doAnalyze Config{..} bks = do
+doAnalyze :: Config -> Text -> [Benchmark] -> IO ()
+doAnalyze Config{..} benchName bks = do
+    packageName <- getProgName
```

Comment:
I do now understand this bit, this actually an executable name, not package name, and each package may contain many such executables. I see that roles of the `getProgName` and `benchName` differ, but I'm not sure how to use them properly, maybe add some documentation about it?